### DISCUSSION TITLE
Option to see the result screen later and smooth caret animation speed

### DISCUSSION THREAD

**Discussion by yekta** [monkeytypegame/monkeytype#1248] [Ideas]
After you complete a typing session a cool result page is shown. It would be nice if we could display this kind of screen later on too. We can see all the data afterwards but not that screen in that form. Also it would be nice if we could adjust the animation speed of the caret. Maybe 3 options: off, normal, fast. I like to have the animation on but it seemed bit too slow to me (not talking about the pace caret, rather the animation time of the caret itself from one letter to another).

This is the screen I want to be able to access at a later time:

![2021-04-13 14 52 00](https://user-images.githubusercontent.com/40706601/114548815-eeb05f00-9c68-11eb-9c44-0248af4dd633.jpg)

These two are the ones I can access at a later time:

<img width="950" alt="Screen Shot 2021-04-13 at 14 59 08" src="https://user-images.githubusercontent.com/40706601/114549003-2b7c5600-9c69-11eb-900b-fbd0b9c69e1f.png">

![download (1)](https://user-images.githubusercontent.com/40706601/114549019-2e774680-9c69-11eb-9ed2-53ee13549a80.png)

**Comment by unrealapex** [comment]
> . It would be nice if we could display this kind of screen later on too. We can see all the data afterwards but not that screen in that form. 

This is not implemented because doing this for all users would fill up the database quickly and bring up the billing price Miodec needs to pay for Firebase.

**Reply by yekta** [reply]
I don't see how that is the case considering almost all the data needed to create that screen is already shown in the 2nd and 3rd photo I shared. Doesn't need to be a 1 to 1 match, what I'm asking for is basically a combination of image 2 and 3 which equates to almost image 1. So it's more about styling things on the front end rather than adding or calling back extra data, the data is already there.

As in, you can access: WPM, accuracy, character count, time etc. at a later time. You can also access the graph at a later time. Which is pretty much what the end screen (image 1) consists of. Those two are just shown in a different way, I was requesting for them to be shown the same way optionally.

**Comment by Miodec** [comment]
Im not sure i see the idea behind this change. All the data is there, just in a different format.

**Reply by yekta** [reply]
Exactly, I just want to be able to see it in the first format at a later time too. We share these images among friends, but image 1 is much better looking one to share compared to the image 2 combined with image 3. If you forget to screenshot the end screen when you complete the test, you can't access it in the same form again (same style). Since that styling already exists, I thought it might not be a big deal to have that option at a later time too.

To reiterate: My request isn't for extra data, it's just for access to a styling option at a later time. That styling option already exists, you just can't access it at a later time.

**Reply by unrealapex** [reply]
I guess it would be helpful if reviewing progress made or milestones.
